app/controllers/events_controller.rb

```diff
@@ -55,6 +68,17 @@ def destroy
 
   private
 
+    def valid_email? email_list
+      @checking_array = email_list.split(/,/).uniq.collect{|x| x.strip || x }
+      valid_email_regex = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
Can you please include positive and negative tests for the email address validation?